### DISCUSSION TITLE
请问nb2这种与各种平台的接入是否安全，是官方认可的接入吗？

### DISCUSSION THREAD

**Discussion by icewizardry** [nonebot/discussions#254] [问答]
会不会因为官方的api变动就失效了

**Comment by yanyongyu** [comment]
nb所有的适配器协议都是各协议官方标准接口定义，至于协议官方的api变动如果是breaking的则需要适配器对应更新

**Reply by icewizardry** [reply]
nb2接入是属于合法的，是不会被各个平台追责的对吧？

**Reply by yanyongyu** [reply]
nb只负责实现通讯协议以及事件处理，至于你协议对接的协议端或平台是否存在违法违规行为与nb无关
